src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -66,9 +66,20 @@
 
 def create_connectedk8s(cmd, client, resource_group_name, cluster_name, correlation_id=None, https_proxy="", http_proxy="", no_proxy="", proxy_cert="", location=None,
                         kube_config=None, kube_context=None, no_wait=False, tags=None, distribution='auto', infrastructure='auto',
-                        disable_auto_upgrade=False, cl_oid=None, onboarding_timeout="600", enable_private_link=None, private_link_scope_resource_id=None):
+                        disable_auto_upgrade=False, cl_oid=None, onboarding_timeout="600", enable_private_link=None, private_link_scope_resource_id=None,
+                        distribution_version=None, azure_hybrid_benefit=None, yes=False, container_log_path=None):
     logger.warning("This operation might take a while...\n")
 
+    # Prompt for confirmation for few parameters
+    if enable_private_link is True:
```

Comment:
We need to check for the SKIP_PROMPT env variable here, without that the runners will be broken.

Reply:
we don't require the SKIP_PROMPT env var anymore. We can just use the --yes flag to skip the prompt message for private link. Let's take this change in the runners too